install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -236,6 +236,8 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: SDS_ENABLED
+            value: "{{ $.Values.global.sds.enabled }}"
```

Comment:
nit: maybe better to replace with `"{{ $.Values.global.sds.enabled }}"` with `{{ $.Values.global.sds.enabled | quote }}`